source/concepts/networking/network-firewall.html.md.erb

```diff
@@ -0,0 +1,31 @@
+---
+owner_slack: "#modernisation-platform"
+title: AWS Network Firewall
+last_reviewed_on: 2022-02-06
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+We currently have setup in our core-network account AWS Network Firewall to monitor and control network traffic on a layer 3 level for transit gateway connection and cross account vpcs to protecte reources
+such as EC2 instances, along with a general aws resources
+
+## What is an AWS network firewall
```

Comment:
This just reads like a copy-paste from an AWS docs page. Does it add any value to a reader, or should we just have a link to the AWS document?

Reply:
yep it was i'll change for a link instead